pkg/nb/rpc.go

```diff
@@ -126,6 +129,7 @@ func NewRPC() *RPC {
 	return &RPC{
 		HTTPClient: http.Client{
 			Transport: util.InsecureHTTPTransport,
+			Timeout: RPCSendTimeout,
```

Comment:
We can start with 2 minutes and see how it goes.
This timeout covers all of the flow: connection, getting the response and reading its body.